### DISCUSSION TITLE
Week 3 Post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#81] [General]
I liked that I could complete the Flexbox Froggy mostly on my own. I think the instructions were easier to follow than the others because the parts I actually need to type were given in the instructions, and nothing was too elaborate. I am, however, currently stuck on the last step. I can't get the row of frogs to align to the right side of the page.

I like that the CSS Cafe Menu shows the results of the coding, like the Froggy one did. That is really helpful to see what the coding is actually doing.

The major con I have still across the board is that most of the instructions aren't obvious enough for a beginner, beginner to follow. The JavaScript tasks are extremely challenging for me, and I often have to look up what I'm missing. I don't think I could complete the JavaScript assignments without YouTube. However, I feel really great when I can complete a task without help, so there is some learning, but generally most of it is still over my head with JavaScript.

**Comment by metasean** [comment]
> I can't get the row of frogs to align to the right side of the page

Will you add a question to the discussions, and see if any of your classmates can help you with this?

> I often have to look up what I'm missing. I don't think I could complete the JavaScript assignments without YouTube

When you run into things that you're having a hard time comprehending, have you tried the "Get Help ▴"?

**Reply by Skye404** [reply]
I have tried the Get Help, but it wasn't as helpful for me the couple times I used it.

**Comment by derekscholes** [comment]
@Skye404 Feel free to reach out on the discussion board. I am trying to look here more often. I can attempt to help with my very limited knowledge or at least give some hints. Also post some of the code you have when you cant figure things out and maybe we can work from there. The Mimo app has really helped me with basic css and web development stuff and it can be done any time you have some free time.

**Reply by Skye404** [reply]
Great, thanks for the info.
